src/main/java/org/eln2/mc/client/screens/CapacitorCellScreen.kt

```diff
@@ -0,0 +1,114 @@
+package org.eln2.mc.client.screens
+
+import com.mojang.blaze3d.systems.RenderSystem
+import com.mojang.blaze3d.vertex.PoseStack
+import net.minecraft.client.gui.components.Button
+import net.minecraft.client.gui.components.EditBox
+import net.minecraft.client.gui.screens.inventory.AbstractContainerScreen
+import net.minecraft.network.chat.Component
+import net.minecraft.network.chat.TextComponent
+import net.minecraft.network.chat.TranslatableComponent
+import net.minecraft.resources.ResourceLocation
+import net.minecraft.world.entity.player.Inventory
+import net.minecraftforge.client.event.ContainerScreenEvent
+import net.minecraftforge.common.MinecraftForge
+import org.eln2.mc.Eln2
+import org.eln2.mc.common.containers.CapacitorCellContainer
+import org.eln2.mc.common.network.Networking
+import org.eln2.mc.common.network.clientToServer.SingleDoubleElementGuiUpdatePacket
+import java.awt.Color
+
+class CapacitorCellScreen(private val container: CapacitorCellContainer, inv: Inventory, name: Component) :
+    AbstractContainerScreen<CapacitorCellContainer>(container, inv, name) {
+
+    private val guiTexture: ResourceLocation = ResourceLocation(Eln2.MODID, "textures/gui/capacitor_gui.png")
+    private val textureWidth = 256
+    private val textureHeight = 128
+    private val guiWidth = 256
+    private val guiHeight = 128
+
+    private var relX: Int = 0
+    private var relY: Int = 0
+
+    private val textboxW = 192
+    private val textboxH = 16
+    private val textboxX = 32
+    private val textboxY = 48
+
+    private val buttonW = 60
+    private val buttonH = 18
+    private val buttonX = 96
+    private val buttonY = 128 - (buttonW / 2)
```

Comment:
Can these be const val? (or otherwise added to a (companion) object so we don't have dozens of instances of them?)